src/jdk.incubator.vector/share/classes/jdk/incubator/vector/X-Vector.java.template

```diff
@@ -645,9 +647,11 @@ public abstract class $abstractvectortype$ extends AbstractVector<$Boxtype$> {
     @ForceInline
     final
     $abstractvectortype$ lanewiseTemplate(VectorOperators.Binary op,
+                                          Class<? extends VectorMask<$Boxtype$>> maskClass,
                                           Vector<$Boxtype$> v) {
```

Comment:
```suggestion
    $abstractvectortype$ lanewiseTemplate(VectorOperators.Binary op,
                                          Vector<$Boxtype$> v) {
```
Remove unused argument by passing `null` to the intrinsic for the `maskClass` argument?